web_widget_dropdown_dynamic/__manifest__.py

```diff
@@ -1,12 +1,12 @@
 # Copyright 2019-2020 Brainbean Apps (https://brainbeanapps.com)
 # Copyright 2020 CorporateHub (https://corporatehub.eu)
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL.html).
 {
     "name": "Dynamic Dropdown Widget",
     "summary": "This module adds support for dynamic dropdown widget",
     "category": "Web",
     "version": "16.0.1.0.0",
```

Comment:
```suggestion
    "version": "16.0.2.0.0",
```

Reply:
Thank you! OCABot will bump the version when merging with the "minor" param.

Reply:
Up to you, I don't have any experience with this, so I believe you know it better.